changelog.d/15886.feature

```diff
@@ -0,0 +1 @@
+Allow configuring the set of workers to proxy outbound federation traffic through via `outbound_federation_restricted_to`.
```

Comment:
Same changelog as https://github.com/matrix-org/synapse/pull/15773 so they merge since we're fixing something that was just merged.